sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/KnownWindowsVirtualMachineImage.java

```diff
@@ -5,7 +5,7 @@
 /** The popular Azure Windows images. */
 public enum KnownWindowsVirtualMachineImage {
     /** Windows 10 2020 H1 Pro */
-    WINDOWS_DESKTOP_10_20H1_PRO("MicrosoftWindowsDesktop", "Windows-10", "20h1-pro"),
+    WINDOWS_DESKTOP_10_20H1_PRO("MicrosoftWindowsDesktop", "Windows-10", "win10-22h2-pro"),
```

Comment:
@weidongxu-microsoft Should we deprecate this enum and provide an alternative major version `WINDOWS_DESKTOP_10_PRO `?

Reply:
Ah, yes, we should do this.

The name of the enum is no longer consistent with its content.